packages/anytype-deb/anytype-deb.pacscript

```diff
@@ -0,0 +1,20 @@
+name="anytype-deb"
+gives="anytype"
+pkgver="0.36.0"
+pkgdesc="Operating environment for the new internet. Anytype is a next generation software that breaks down barriers between applications, gives back privacy and data ownership to users."
```

Comment:
Too long of a description. Please make a newline before the second sentence so it turns into a long description.
```suggestion
pkgdesc="Operating environment for the new internet
Anytype is a next generation software that breaks
down barriers between applications, gives back privacy
and data ownership to users."
```